### PR TITLE
Switch to mkdocs-material insiders repo

### PR DESCRIPTION
**Pull request by grugnog**
----
:books: Documentation preview :books:: https://civicactions-handbook--1044.org.readthedocs.build/en/1044/

### CODE REVIEW

**Comment by grugnog** [comment]
TODO: Update path to file in Docker Hub after this is merged.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 version: 2
 updates:
   - package-ecosystem: "pip"
```
Comment (dmundra) [inline .github/dependabot.yml:6]: I need to readup on poetry feature but want to comment whether that needs to be specified here instead of pip?

Comment (grugnog) [reply .github/dependabot.yml:6]: This ecosystem is correct for poetry also, according to the docs

Comment (dmundra) [reply .github/dependabot.yml:6]: Excellent! Thanks!

**Review by dmundra — APPROVED** [review]
I think this is looking good to ship!

**Comment by ContentBrewmaster** [comment]
[Sponsor level for Material theme](https://trello.com/c/tA65MGYm/335-sponsor-level-for-material-theme)

**Pull request merged**